dagster_sqlmesh/controller/base.py

```diff
@@ -425,10 +427,12 @@ def setup(
         context_factory: ContextFactory[ContextCls],
         gateway: str = "local",
         log_override: logging.Logger | None = None,
+        translator: SQLMeshDagsterTranslator = SQLMeshDagsterTranslator(),
```

Comment:
```suggestion
        translator: SQLMeshDagsterTranslator | None = None,
```

```suggestion
        translator: SQLMeshDagsterTranslator | None = None
```

Do this and set the value to the instance later in the factory method.